composer.json

```diff
@@ -12,7 +12,7 @@
         }
     ],
     "require": {
-        "wp-cli/wp-cli": "^2.5"
```

Comment:
In this file, we can add the new commands further below in the `commands` section.

This way they will be picked up when regenerating the readme (happens automatically but you can also run `wp scaffold package-readme`)

Reply:
I've added the new commands.